cypress/component/ResultCard.cy.tsx

```diff
@@ -24,6 +30,7 @@ describe('ResultCard', () => {
         datasetTotalSubjects={props.datasetTotalSubjects}
         numMatchingSubjects={props.numMatchingSubjects}
         imageModals={props.imageModals}
+        pipelines={props.pipelines}
```

Comment:
please also update test description to include "pipeline"
Generally: this test is a bit "Everything", we may want to start testing things individually and use `beforeEach` to keep the setup the same. Otherwise the test descriptions become half of an essay 
and the tests meaningless